setup.py

```diff
@@ -64,7 +64,8 @@
         "scikit-learn",
         "scipy",
         "pytest",
-        "transformers>=3.4,<3.6",
+        "transformers>=3.4,<4.1",
```

Comment:
I think we'll need to pin this to 

```suggestion
        "transformers>=4.0,<4.1",
```

since some imports have changed, right?

Reply:
Ah, yes, you're right.